content/en/docs/OSSMC/users-guide.md

```diff
@@ -78,6 +78,19 @@ If you hover over a cell in a heatmap, a tooltip will give some details on the c
 
 ![Workload: Traces Heatmap](/images/documentation/installation/installation-guide/29-workload-traces-heatmap.png)
 
+When the OpenShift tracing UI plugin is enabled, Kiali will try to auto discover the plugin settings and the `View in Tracing` Kiali link will redirect to the plugin (from Kiali 2.8.0). 
+If the plugin config needs to be adjusted, the following settings should be updated:
+
+```yaml
+{
+  "observability": {
+    "instance": "sample",
+    "namespace": "tempo",
+    "tenant": "default"
+  }
+}
+```
```

Comment:
It is not clear where the user needs to set this yaml configuration. This isn't the Kiali CR. Tell the user where this configuration is. I'm not sure myself :)